pybliometrics/scopus/utils/startup.py

```diff
@@ -1,21 +1,81 @@
-import configparser
+from configparser import ConfigParser
 from collections import deque
 from pathlib import Path
+from typing import List, Dict, Optional, Type
+import os
+from warnings import warn
 
 from pybliometrics.scopus.utils.constants import CONFIG_FILE, RATELIMITS
 from pybliometrics.scopus.utils.create_config import create_config
 
-# Read/create config file (with fixture for RTFD.io)
-config = configparser.ConfigParser()
-config.optionxform = str
-try:
-    if not CONFIG_FILE.exists():
-        config = create_config()
+CUSTOM_DIR = None
```

Comment:
What is the purpose of this variable?

Reply:
It contains the path to the config file if the user defined one. It is used to check if the user passed a path or not.